### ISSUE TITLE
Oh no! An error occured: ENOENT: no such file or directory

### ISSUE THREAD

**Issue by okkimus** [jpb06/jest-badges-action#48]
Hi!

I am trying to add this action to my project. Repo is [here](https://github.com/okkimus/TDD-Lister).

I'm generating the needed report json. In the action, I can see that the file exists and even `cat` the contents. But when this action starts generating the badges, it fails to find the file.

At first I was using relative paths when the action was failing in earlier step. But then I decided to just use the absolute paths. However, still the action can't find the report.

You can see my action [here](https://github.com/okkimus/TDD-Lister/blob/main/.github/workflows/node.js.yml) and example of failed run [here](https://github.com/okkimus/TDD-Lister/actions/runs/4230233846).

If you have an idea what could cause this, I'm also happy to help. (No experience in creating actions for marketplace, but happy to learn more on it.)

**Comment by jpb06** [comment]
Hey. 

That's odd. I forked your repo to take a look!

**Comment by jpb06** [comment]
Looks like it's linked with the usage of `working-directory` that changes `pwd`. `readJson` works within action code but doesn't in the dependency `node-jest-badges`. 

Hang in there, I will try to debug this locally using `act` to find the issue.

**Comment by okkimus** [comment]
I saw you debugging this. Thanks for that.

I tried also to just move the whole `backend` to the root of the repo and updated the paths and removed the `working-directory` setting etc. but that didn't work either. But when I removed the extra params (`coverage-summary-path` and `output-folder`) it worked. I think something funky is going on how those params are handled.

I was browsing through the source code in the morning to see if there was something obvious, but couldn't spot any weirdness with a quick look at it.

**Comment by jpb06** [comment]
I think I found the issue. `outputPath` was improperly set and we also checked if badges had changed.... after generated them.

https://github.com/jpb06/jest-badges-action/pull/50 should fix that.

**Comment by jpb06** [comment]
Released as [1.9.11](https://github.com/jpb06/jest-badges-action/releases/tag/v1.9.11).

[Validated](https://github.com/jpb06/TDD-Lister/actions/runs/4245687473) using these params:

```yaml
     [...]

      - name: Generating coverage badges for backend
        uses: jpb06/jest-badges-action@v1.9.11
        with:
          coverage-summary-path: ./src/backend/coverage/coverage-summary.json
          output-folder: ./src/backend/badges
```

Let me know how it works for you 🙇🏻

**Comment by okkimus** [comment]
Excellent, this works now! Much appreciated, the badges look cool 😎  Have a nice week! 👍
